app/src/main/res/layout/options_privacy.xml

```diff
@@ -59,7 +59,7 @@
                     app:description="@string/security_options_tracking_protection" />
 
                 <TextView
-                    android:id="@+id/setting_description"
+                    android:id="@+id/permissionsTitle"
```

Comment:
I was surprised this didn't cause side effects in any of these views:

```
% ack setting_description --java app/src
app/src/common/shared/org/mozilla/vrbrowser/ui/views/settings/SwitchSetting.java
40:        mText = attributes.getString(R.styleable.SwitchSetting_description);
53:        mSwitchDescription = findViewById(R.id.setting_description);

app/src/common/shared/org/mozilla/vrbrowser/ui/views/settings/ButtonSetting.java
32:        mDescription = attributes.getString(R.styleable.ButtonSetting_description);
44:        TextView description = findViewById(R.id.setting_description);

app/src/common/shared/org/mozilla/vrbrowser/ui/views/settings/RadioGroupSetting.java
43:        mDecription = attributes.getString(R.styleable.RadioGroupSetting_description);
69:        mRadioDescription = findViewById(R.id.setting_description);

app/src/common/shared/org/mozilla/vrbrowser/ui/views/settings/SingleEditSetting.java
43:        mDescription = attributes.getString(R.styleable.EditSetting_description);
56:        mDescriptionView = findViewById(R.id.setting_description);
```

Reply:
In the instance that I changed, the previous id was just a default, and this didn't affect anything because nowhere in the code was that id being queried, but to access to format the string a non-default id was required. In the other four instances you identify above, the views are acting as templates and so use of a generic id enables the desired behaviour.